scripts/vif-real

```diff
@@ -240,11 +240,11 @@ remove_from_bridge()
         # port.  Use --if-exists to suppress the error that would otherwise
         # arise in that case.
         $vsctl --timeout=30 -- --if-exists del-port $dev
-        xenstore-rm ${PRIVATE}/"$TYPE"-ofport
+        xenstore-rm "${HOTPLUG}/$TYPE-ofport"
```

Comment:
Here, above, and below, I'd write `"${HOTPLUG}/${TYPE}-ofport"`. It does work as it is, however,